mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,160 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+'@media/dynamic-range' represents the combination of brightness and color
+depth.
+
+<dl dfn-type=value dfn-for="@media/dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered high
+		* color depth is greater than 24 bit
+		* has a <a>high contrast ratio</a>
+	<dt><dfn>standard</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* max brightness is considered standard
+		* color depth is 24 bit or 8 bit per color component of RGB
+		* has a <a>low contrast ratio</a>
+</dl>
+
+<h3 id="contrast-brightness-of-display">
+	Determining contrast and brightness of display</h3>
+
+This text is non-normative. There currently is no agreed upon standardization of
+<dfn export>high contrast ratio</dfn>, <dfn export>low contrast ratio</dfn>,
+max brightness or low brightness as it relates to displays. As such, the
+determination for '@media/dynamic-range' and '@media/video-dynamic-range' will
+be defined by the user agent.
+
+<h3 id="video-prefixed-properties">
+Video Prefixed Properties scenarios</h3>
+
+    Some user agents, including many TVs, render video and graphics in two
+	separate "planes" (bi-plane) with distinct screen characteristics. A set of
+	video-prefixed features is provided to describe the video plane.
+
+	Any bi-plane implementation **MUST** return values based on the video plane
+	for the following features: '@media/video-color-gamut'; '@media/video-width';
+	'@media/video-height'; '@media/video-resolution'; '@media/video-dynamic-range'.
+	All other features **MUST** return values based on the graphics plane.
+
+    Non bi-plane implementations **MUST** return the same values for
+	video-prefixed features and their non-prefixed counterparts.
+
+<h3 id="video-color-gamut">
+Video Color Display Quality: the 'video-color-gamut' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: video-color-gamut
+	Value: srgb | p3 | rec2020
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+The '@media/video-color-gamut' media feature describes the approximate range of colors
+	that are supported by the UA and output device's video plane.
+	That is, if the UA receives content with colors in the specified space
+	it can cause the output device to render the appropriate color,
+	or something appropriately close enough.
+
+Value and color space definitions are the same as <a href="#color-gamut">color-gamut</a>
+
+<h3 id="video-dynamic-range">
+Video Dynamic Range: the 'video-dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: video-dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+'@media/video-dynamic-range' has the same values and definitions as '@media/dynamic-range',
```

Comment:
I would change to:
'''
'@media/video-dynamic-range' represents the combination of max brightness, color depth, and contrast ratio that are supported by the UA and output device's video plane.

Supported values are the same as <a href="#dynamic-range">dynamic-range</a>.
'''